content/en/docs/reference/kubectl/docker-cli-to-kubectl.md

```diff
@@ -7,7 +7,7 @@ reviewers:
 ---
 
 <!-- overview -->
-You can use the Kubernetes command line tool kubectl to interact with the API Server. Using kubectl is straightforward if you are familiar with the Docker command line tool. However, there are a few differences between the docker commands and the kubectl commands. The following sections show a docker sub-command and describe the equivalent kubectl command.
+You can use the Kubernetes command line tool `kubectl` to interact with the API Server. Using kubectl is straightforward if you are familiar with the Docker command line tool. However, there are a few differences between the Docker commands and the kubectl commands. The following sections show a docker sub-command and describe the equivalent `kubectl` command.
```

Comment:
We refer to the Docker technology in general as `"Docker"`, and the `docker` command line as `"docker"`.

Reply:
Thanks a lot for the explanation. I fixed it.